rendering_resource_manager_service/session/management/session_manager_settings.py

```diff
@@ -56,7 +56,7 @@
                       '/var/log/vws'
 SLURM_ERR_FILE = 'err.log'
 SLURM_OUT_FILE = 'out.log'
-SLURM_ALLOCATION_TIMEOUT = 20
+SLURM_ALLOCATION_TIMEOUT = 10
```

Comment:
Actually, this value is overridden by the deployment process (like for all other Slurm related variable). I reduced the value from 20 to 10 when I was testing the feature. I can set it back to 20 but again, the deployment process will have the final word.